### ISSUE TITLE
Prettytime.formatUnrounded giving - with russian string

### ISSUE THREAD

**Issue by B0pol** [ocpsoft/prettytime#182]
<img src="https://user-images.githubusercontent.com/800970/71483213-d02a1880-2817-11ea-9cb7-b7ec64cffb5d.jpg" width="200">
You can see the date of the videos at the rigth.
The screenshot is from NewPipe, using Prettytime.formatUnrounded(Calendar then) function.

You can reproduce it with by adding this in [PrettyTimeI18n_RU_Test.java](https://github.com/ocpsoft/prettytime/blob/master/core/src/test/java/org/ocpsoft/prettytime/PrettyTimeI18n_RU_Test.java)
```java
@Test
public void testCalculatePreciseDuration2() {
    PrettyTime prettyTime = new PrettyTime();
    prettyTime.clearUnits();
    Minute minutoj = new Minute();
    prettyTime.registerUnit(minutoj, new ResourcesTimeFormat(minutoj));
    Assert.assertEquals("40 minutes ago",prettyTime.formatUnrounded(
      prettyTime.calculatePreciseDuration(new Date(new Date().getTime() - 40 * 60 * 1000 - 40 * 1000))));
}
```
Output:
```
org.junit.ComparisonFailure: 
Expected :40 minutes ago
Actual   :-40 минут назад
```
As you can see, there is that ``-``

**Comment by lincolnthree** [comment]
Hi, thanks for submitting this issue.

Apologies for the delay in responding. I haven't had a chance to evaluate your test case yet (thanks for attaching a reproduction). I will try to get to it soon, but if you find the root cause before then, pull requests are very welcome!

Thanks and sorry again for the wait,
~Lincoln

**Comment by sryze** [comment]
Hello, I opened a pull request to fix this issue in NewPipe, please take a look: https://github.com/ocpsoft/prettytime/pull/189

**Comment by B0pol** [comment]
Fixed by #189
